src/intel_driver.h

```diff
@@ -111,6 +111,18 @@ extern uint32_t g_intel_debug_option_flags;
 #define VA_INTEL_DEBUG_OPTION_BENCH     (1 << 1)
 #define VA_INTEL_DEBUG_OPTION_DUMP_AUB  (1 << 2)
 
+#define LOCAL_I915_CONTEXT_PARAM_WATCHDOG 0x7
+#define LOCAL_I915_EXEC_VEBOX             (4<<0)
+#define LOCAL_I915_EXEC_VCS2              (5<<0)
```

Comment:
Why is LOCAL_I915_EXEC_VCS2 needed? we don't use 5 (LOCAL_I915_EXEC_VCS2) to indicate vcs2 ring.

Reply:
Also, i915 does not and will not support this flag. If you were thinking of adding support for new engines you will have to make use of the context engine map feature (or the Virtual Engine). Neither is upstream yet.

Reply:
@tursulin which flag are u referring to? the one being suggested by @xhaihao  (I915_EXEC_BSD_RING2 | I915_EXEC_BSD? I tried on my testbed and it both compiles and executes correctly on my testbed.

Reply:
@xhaihao please take a look at updated patch on this branch.